src/app/services/ws.service.ts

```diff
@@ -350,4 +354,19 @@ export class WebSocketService {
       this.window.location.reload();
     });
   }
+
+  private getLoggedInUserInformation(): void {
+    let authMeUser: DsUncachedUser;
+
+    this.call('auth.me').pipe(
+      switchMap((loggedInUser: DsUncachedUser) => {
+        authMeUser = loggedInUser;
```

Comment:
No point in having this variable and assignment.

Reply:
we need to keep data from `auth.me` and then I merge it with `user.query` data.